src/ray/protobuf/common.proto

```diff
@@ -310,14 +310,17 @@ message TaskSpec {
   string concurrency_group_name = 24;
   // Whether application-level errors (exceptions) should be retried.
   bool retry_exceptions = 25;
+  // A serialized exception allowlist that takes a frontend-language
```

Comment:
This needs to be updated, no longer a function.

Reply:
Hmm this one seems to be up-to-date already.